spec/signable/query/client_spec.rb

```diff
@@ -0,0 +1,259 @@
+require "spec_helper"
+
+describe Signable::Query::Client do
+  include_context 'when signable is configured'
+
+  EXISTENT_FINGERPRINT = '83ed33d1844c54a053233d00e6ab6e33'.freeze
```

Comment:
This is the fingerprint of an envelope we created on Signable to use with our tests.